### ISSUE TITLE
bugs occur when rotating the screen

### ISSUE THREAD

**Issue by littlemozart** [kitasuke/PagingMenuController#345]
It seems that everything works well when the screen is in portrait. 
![bea2e6cf-dced-4a37-9dc3-f0baa0df0e85](https://cloud.githubusercontent.com/assets/18299148/24936130/4293e966-1f59-11e7-93bb-5ba91c3e0498.png)
But once I rotate the screen, the menu bar will look higher than usual and the content view will be cut.
![c7552d17-9ee0-4c59-bcb6-68c4f224fd49](https://cloud.githubusercontent.com/assets/18299148/24936139/4f2510e2-1f59-11e7-9c83-4cfe96362c03.png)

**Comment by junestchick** [comment]
Yes, the same as me. I put a collectionView in first segment and when rotating device, the size of cell is not updated. Any suggest ?

**Comment by UmaMadhavi** [comment]
when i add tableview i am getting crash . can u know why this happens
